go.mod

```diff
@@ -143,4 +146,10 @@ replace github.com/opencontainers/image-spec => github.com/opencontainers/image-
 
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.3
 
-replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
+// replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
+
+replace github.com/cosmos/ibc-go/v3 => github.com/cosmos/ibc-go/v3 v3.0.0-alpha1.0.20220510152759-dcd0681d8f07
+
+replace github.com/CosmWasm/wasmd => github.com/devashishdxt/wasmd v0.27.1-0.20220530042031-4126e5b964df
```

Comment:
could you update it to this fork: https://github.com/crypto-org-chain/wasmd ?

Reply:
Done